### PR TITLE
Fixes looking around

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

**Comment by fstagni** [comment]
Alright?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -14,6 +14,21 @@ defaults:
     shell: bash -el {0}
 
 jobs:
+
+  shellcheck:
```
Comment (chrisburr) [inline .github/workflows/main.yml:18]: Is this worth having? I think we only have the run-local.sh file

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # See https://pre-commit.com for more information
 # See https://pre-commit.com/hooks.html for more hooks
 default_language_version:
-  python: python3
+  python: python3.11
```
Comment (chrisburr) [inline .pre-commit-config.yaml:4]: I think we should avoid specifying the version here so it's one less place to keep track of

```suggestion
  python: python3
```

File: .github/workflows/main.yml (hunk 1 above)
Comment (fstagni) [reply .github/workflows/main.yml:18]: It does not hurt

**Comment by fstagni** [comment]
OK now?

**Review by chrisburr — APPROVED** [review]

**Pull request merged**